src/containers/Token/index.js

```diff
@@ -11,6 +11,9 @@ import NoMatch from '../NoMatch';
 import './styles.css';
 import { analytics, ANALYTIC_TYPES, NOT_FOUND, BAD_REQUEST } from '../shared/utils';
 
+const MODE = process.env.REACT_APP_ENVIRONMENT;
+const IS_MAINNET = MODE === 'mainnet';
```

Comment:
```suggestion
const IS_MAINNET = process.env.REACT_APP_ENVIRONMENT === 'mainnet';
```
Since `MODE` isn't used anywhere as far as I can tell, it seems cleaner to do it in one line.